css/danti.html

```diff
@@ -0,0 +1,127 @@
+
+            <nav class="nuv_bar">
+                <ul class="nav_ul_p2">
+                    <li>Meet Karma</li>
+                    <li>How it Work</li>
+                    <li><a href="../index.html"> Store</a></li>
+                    <li>Blog</li>
+                    <li>Help</li>
+                    <li>Login</li> 
+                </ul>
+            </nav>
+        </div>
+
+        <div class="img_contener">
+            <img src="../level-2/store-image_by-andrew-neel-unsplash.jpg" alt="">
+        </div>
+
+        <!-- <div class="form">form</div> -->
+        <div class="form">
+            <form action="#">
+                <div>
+                    <h2 class="title">Order your Karma wifi <br>divice today!</h2>
+                </div>
+                <div class="first_last">
+                    <div class="first_name">
+                        <label for="name">First name *</label><br>
+                        <input type="text" id="name_input" name="name" value=""/>
+                    </div>
+        
+                    <div class="last_name">
+                        <label for="last_name">Last name *</label><br>
+                        <input type="text" id="last_name_input" name="last_name" value=""/>
+                    </div>
+                </div>
+                
+                <div class="address">
+                    <label for="adress">Address *</label><br>
+                    <input type="text" id="addres_input" name="address" value=""/>
+                </div>
+    
+                <div class="address_2">
+                    <label for="address_2">Address 2</label><br>
+                    <input type="text" id="address_input_2" name="address_2" value="">
+                </div>
+                <div class="city_postcode">
+                    <div class="city_city">
+                        <label for="city">City *</label><br>
+                        <select id="city" name="city">
+                            <option class="city_input" value="Select" selected aria-placeholder="Select your city..."></option>
+                            <option value="Eri">Eritria</option>
+                        </select>
+                    </div>
+                    <div class="postcode">
+                        <label for="city">Post code *</label><br>
+                        <input type="text" id="postcode_input" name="city" value=""/>
+                    </div><br>
+                    </div>
+                    <div>
+                        <label for="color" class="color">Select a colour</label><br>
+                        
+                        <div class="color_radio">
+                            <div class="karma_orange">    
+                                <input id="orange" class="orange" type="radio" name="size" value="Karma orange" />
+                                <div class="lab_1"><label for="small">Karma orange</label></div>
```

Comment:
Nice work remembering to add `for` attributes to your labels for accessibility :tada:  In this case, I think you just forgot to update the value of the attribute to match the input:

```suggestion
                                <div class="lab_1"><label for="orange">Karma orange</label></div>
```